### PR TITLE
Update zh_CN.lang

### PR DESCRIPTION
**Pull request by meitangdehulu**
Fix some errors and proofreading

### CODE REVIEW

**Comment by ThatGamerBlue** [comment]
#4104

**Comment by meitangdehulu** [comment]
This is the latest

**Comment by meitangdehulu** [comment]
This is the latest.



________________________________
发件人: ThatGamerBlue <notifications@github.com>
发送时间: 2016年12月31日 14:23
收件人: aidancbrady/Mekanism
抄送: meitangdehulu; Author
主题: Re: [aidancbrady/Mekanism] New (#4106)


#4104<https://github.com/aidancbrady/Mekanism/pull/4104>

―

**Comment by 61352151511** [comment]
Well, between #4104 and this, it seems #4104 just had the English terms put into google translate and the results were copied into the translation files. So I would vote for this PR knowing google translate is wrong most of the time.

File: src/主要/资源/资产/mekanism/朗/zh_CN.lang [hunk 1]
```diff
@@ -779,68 +806,69 @@ tooltip.BoilerCasing=一种耐压方块,用于制作锅炉结构的外壳
...
+tooltip.SolarGenerator=一台使用太阳能量发电的发电机.
+tooltip.GasGenerator=一台利用不同的气体供能的发电机.
+tooltip.BioGenerator=一台利用燃烧有机材料产生能量的发电机.
+tooltip.AdvancedSolarGenerator=一台直接吸收大部分太阳光产生能量的高级发电机.
+tooltip.WindTurbine=一台利用风能产生能量的发电机, 高度越高效率越高.
 tooltip.TurbineRotor=放置在工业涡轮中的钢制转子
 tooltip.RotationalComplex=放置在涡轮转子上方的转化元件,能够将转子的动能转化.
 tooltip.ElectromagneticCoil=一块类似磁铁的装置,能使得动能在其中转化为电磁能,它必须放置在旋钮装置的上方并保证它们是相连的.
-tooltip.TurbineCasing=用于工业涡轮的耐压外壳
+tooltip.TurbineCasing=用于工业涡轮的耐压 外壳
```
Comment (TartaricAcid) [inline src/主要/资源/资产/mekanism/朗/zh_CN.lang]: Please delete the space

**Comment by TartaricAcid** [comment]
please fix all the mistake.

**Comment by meitangdehulu** [comment]
Welcome your visit.
http://www.mcbbs.net/thread-654082-1-1.html
[1.10.2-1.7.10][Mekanis ――通用机械][自带中文][V9]强大的工业玄学&nbsp;-&nbsp;Minecraft(我的世界)中文论坛 - Minecraft资讯|MOD|联机|皮肤|材质|作品<http://www.mcbbs.net/thread-654082-1-1.html>
www.mcbbs.net
[1.10.2-1.7.10][Mekanis ――通用机械][自带中文][V9]强大的工业玄学 -



________________________________
发件人: 没糖的 葫芦
发送时间: 2016年12月31日 14:48
收件人: aidancbrady/Mekanism
主题: 答复: [aidancbrady/Mekanism] New (#4106)


This is the latest.



________________________________
发件人: ThatGamerBlue <notifications@github.com>
发送时间: 2016年12月31日 14:23
收件人: aidancbrady/Mekanism
抄送: meitangdehulu; Author
主题: Re: [aidancbrady/Mekanism] New (#4106)


#4104<https://github.com/aidancbrady/Mekanism/pull/4104>

―

**Comment by 3TUSK** [comment]
...There is no need to acquire the permission. Mekanism is licensed under MIT.

**Pull request closed without merging**